src/Symfony/Component/Cache/Marshaller/DefaultMarshaller.php

```diff
@@ -21,15 +21,17 @@
 class DefaultMarshaller implements MarshallerInterface
 {
     private $useIgbinarySerialize = true;
+    private $debug;
 
-    public function __construct(bool $useIgbinarySerialize = null)
+    public function __construct(bool $useIgbinarySerialize = null, bool $debug = false)
```

Comment:
I'd rather name it `$throwOnSerializationFailure` than `$debug`, to make it more clear what it's effect is (which does not prevent using `%kernel.debug%` as the value injected in it)

Reply:
works for me, updated